processing_code/Redesign_scripts/Redesign-knit-RMD-2-MD_NDSRepo.R

```diff
@@ -22,32 +22,34 @@ require(markdown)
 # You can choose a high-level directory and this script will search
```

Comment:
This is fine for now, but I generally do not include the Redesign-kniw-RMD-2-MD-NDSRepo.R in my PRs because those changes are specific to your computer and this particular update